tests/test_spec.py

```diff
@@ -306,7 +310,7 @@ def test_calibrate_dispersion_model():
     psf_centroid.xfit_err = psf_centroid.xfit_err[:-1]
     psf_centroid.yfit_err = psf_centroid.yfit_err[:-1]
     
-    disp_model = steps.calibrate_dispersion_model(psf_centroid, prism = 'PRISM3')
+    disp_model = steps.calibrate_dispersion_model(psf_centroid)
```

Comment:
The relative image shifts between the sub-band filters are here accounted for in test_calibration_dispersion_model(), but not inside the calibration function. Instead, the image shifts will need to be considered an input to calibration_dispersion_model().